packages/tupaia-web/src/features/Map/MapOverlays/DataVisualsLayer.tsx

```diff
@@ -48,45 +48,46 @@ export const DataVisualsLayer = ({
   }
 
   return (
-    <LayerGroup>
-      {processedMeasureData.map(measure => {
-        const { region, organisationUnitCode: entity, color, name } = measure;
-        if (region) {
+    <>
+      <LayerGroup>
+        {processedMeasureData.map(measure => {
+          if (measure.region) {
```

Comment:
Could be nice to go back to the destructuring of before. So this file should be unchanged (except for any possible reusable hooks that get used instead)